go-controller/pkg/ovn/pods.go

```diff
@@ -204,6 +215,15 @@ func (oc *Controller) addRoutesGatewayIP(pod *kapi.Pod, podAnnotation *util.PodA
 	return nil
 }
 
+func (oc *Controller) getRoutingExternalGWsAnnotation(ns string) (net.IP, error) {
```

Comment:
I don't like the name of this. it is not getting the annotation

Reply:
I cargo culted from getHybridOverlayExternalGwAnnotation, but you are right, we return a net.IP thus we should remove
the 'annotation' from this accessor methods.